main/settings.py

```diff
@@ -732,6 +768,12 @@
     },
 }
 
+if ENABLE_WAYBACK_TASKS:
```

Comment:
@ibrahimjaved12, as discussed, this feature flag in the environment requires restarting web workers. This kind of arrangement is not suitable for RC or Production. Please remove this check and always register the tasks.
You can control the logic of tasks to be empty with Posthog feature flag.